test/core/runtime/wavm/runtime_paths.hpp

```diff
@@ -0,0 +1,415 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_RUNTIME_PATHS_HPP
+#define KAGOME_RUNTIME_PATHS_HPP
+
+constexpr std::string_view kBasePath =
+    "./kusama-para-code/";
+
+constexpr std::string_view kRococoParachains[] = {
+    "01165a749e9bb0c9cdceec7745cdf2693fee68bce0a4de583488c4af5643bbdd.wasm",
```

Comment:
Do we need to distribute this .wasm files as well?

Reply:
Not really. Added `parachain_runtimes/README.md` and required scripts to produce those files locally on demand.